pyproject.toml

```diff
@@ -161,6 +161,10 @@ indent-style = "space"
     "F841",   # Local variable {variable} is assigned to but never used
     "N999",   # Invalid module name
 ]
+"src/crawlee/_utils/models.py" = [
+    "FA100",  # In order to be able to use | instead of Union for type hints in Pythons<3.10 we need to do from __future__ import annotations. This import can break pydantic in unxpecetd ways.
+]
+
```

Comment:
I'd rather put that at the start of the file... this seems kinda fragile.

Reply:
Pydantic people solved the issue I reported. So I will just wait for thew new release and then this will not longer be needed.